vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql

```diff
@@ -95,6 +95,7 @@ CREATE VIEW full_proposals_info
 AS
 SELECT
     proposals.*,
+    coalesce(reviews_count, 0) as reviews_count,
```

Comment:
`coalesce` made me look into the docs for the semantics.
```suggestion
    ifnull(reviews_count, 0) as reviews_count,
```

Reply:
Now that I check on it sqlite support `ifnull`, but for some reason in the docs I first checked it forward me to `coalesce`